test/development/acceptance-app/hydration-error.test.ts

```diff
@@ -834,41 +834,39 @@ describe('Error overlay for hydration errors', () => {
     if (isTurbopack) {
       expect(fullPseudoHtml).toMatchInlineSnapshot(`
         "...
-          <NotFoundErrorBoundary pathname="/" notFound={[...]} notFoundStyles={[...]} asNotFound={undefined} missingSlots={Set}>
-            <RedirectBoundary>
-              <RedirectErrorBoundary router={{...}}>
-                <InnerLayoutRouter parallelRouterKey="children" url="/" tree={[...]} childNodes={Map} segmentPath={[...]} ...>
-                  <ClientPageRoot props={{params:{}, ...}} Component={function Page}>
-                    <Page params={{}} searchParams={{}}>
+          <RedirectBoundary>
+            <RedirectErrorBoundary router={{...}}>
+              <InnerLayoutRouter parallelRouterKey="children" url="/" tree={[...]} childNodes={Map} segmentPath={[...]} ...>
+                <ClientPageRoot props={{params:{}, ...}} Component={function Page}>
+                  <Page params={{}} searchParams={{}}>
+                    <div>
                       <div>
                         <div>
                           <div>
-                            <div>
-                              <Mismatch>
-                                <p>
-                                  <span>
-                                    ...
-        +                            client
-        -                            server"
+                            <Mismatch>
+                              <p>
+                                <span>
+                                  ...
+        +                          client
+        -                          server"
       `)
     } else {
       expect(fullPseudoHtml).toMatchInlineSnapshot(`
         "...
-          <NotFoundErrorBoundary pathname="/" notFound={[...]} notFoundStyles={[...]} asNotFound={undefined} missingSlots={Set}>
```

Comment:
Not sure why this frame disappeared. @huozhi do we slice something off from the top of the stack?

Reply:
It's all what react logged in the console, we didn't trim anything off from it

Reply:
Odd, will keep an eye on it but I don't think it's urgent since it's a framework frame anyway. I asked @sebmarkbage if he maybe has a clue.